edi_notification_oca/components/listener.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _
+
+from odoo.addons.component.core import Component
+
+
+class EdiNotificationListener(Component):
```

Comment:
what is the advantage of using a component here instead of simply overriding `_notify_error` on `edi.exchange.record`?